lib/asm.js

```diff
@@ -163,12 +163,23 @@ function AsmParser(compilerProps) {
         const stdInLooking = /.*<stdin>|^-$|example\.[^/]+$|<source>/;
         var endBlock = /\.(cfi_endproc|data|text|section)/;
         let source = null;
+        var startAppBlock = /.*\#APP.*/;
```

Comment:
Just a FYI, we're slooowly moving towards using `const` and `let` instead of `var`. They have (slightly) better semantics. These could both be `const`, and the `inCustomAssembly` could get a `let`, in this case.